os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -789,12 +789,7 @@ int8_t cmd_wifi_off(void)
 		}
 	}
 	RTW_API_INFO("\n\rWIFI Mode Change instead of WIFI reload\r\n");
-
-#if 0
-	if (!wifi_off())
-		return 0;
-	return -1;
-#endif
+	return 0;

```

Comment:
How about moving this `return` after `#endif` in line 793?
Because for `##if CONFIG_ENABLE_P2P` condition, there is no return statement.